site/gatsby-site/src/components/styles/Form.js

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
+import { Form, Button } from 'react-bootstrap';
+
+// Reference:
```

Comment:
I think this has moved beyond the original reference and could likely be removed here.